ui/app/pages/swaps/swaps-gas-customization-modal/swaps-gas-customization-modal.container.js

```diff
@@ -40,6 +40,7 @@ const mapStateToProps = (state) => {
     extraInfoRow = null,
     initialGasPrice,
     initialGasLimit,
+    minimumGasLimit,
```

Comment:
The `modalProps` here appears to be _optional_ (because there is an empty object fallback on the line below this), but `minimumGasLimit` is a _mandatory_ prop for the component.

Should `minimumGasLimit` be made optional, or a default value used somewhere? Or should `modalProps` be considered mandatory, and that fallback removed? :thinking:  The latter is my guess.

Reply:
The fallback object should be removed and modalProps should be considered mandatory. Right now, this modal is only used from one location, where modal props are passed. I can't think of another possible use case for this modal at this time.

I have removed the fallback in 6cede74c2